src/app/work-item/work-item.service.ts

```diff
@@ -54,26 +66,35 @@ export class WorkItemService {
       .get(url, { headers: this.headers })
       .toPromise()
       .then(response => {
+        // Build the user - id map
         this.buildUserIdMap();
+        let wItems: WorkItem[];
         if (process.env.ENV == 'inmemory') {
           // Exclusively for in memory testing
```

Comment:
Yeah, true. We don't need that anymore. Just bear with it for couple more days. We are going to get rid off all in-memory stuffs. So, I just left it as it is fir now.